pages/linux/ksvgtopng5.md

```diff
@@ -0,0 +1,7 @@
+# ksvgtopng5
+
+> Ksvgtopng5: Convert SVG files to PNG format.
+
+- Convert a SVG file to PNG:
+
+`ksvgtopng5 {{width}} {{height}} {{full_path/to/file}} {{output_filename}}`
```

Comment:
path/to/file is our usual convention that we use. It indicates both full and relative file paths.

Also, does the output file need to have a ".png" extension? Maybe good to write the example as path/to/input.svg and output.png.

Reply:
the source file should have an absolute path